client/reader/following/main.jsx

```diff
@@ -96,12 +96,12 @@ const FollowingStream = ( props ) => {
 			<SectionHeader label={ translate( 'Followed Sites' ) }>
 				{ config.isEnabled( 'reader/seen-posts' ) && ! props.hasUnseen && (
 					<Button compact onClick={ markAllAsUnSeen }>
-						{ translate( 'Mark All as Unseen' ) }
+						{ translate( 'Mark All as unseen' ) }
 					</Button>
 				) }
 				{ config.isEnabled( 'reader/seen-posts' ) && props.hasUnseen && (
 					<Button compact onClick={ markAllAsSeen }>
-						{ translate( 'Mark All as Seen' ) }
+						{ translate( 'Mark All as seen' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Mark all as seen' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/unseen-for-subsections)